google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Mutation.java

```diff
@@ -44,17 +44,29 @@
   @InternalApi("Visible for testing")
   static final int MAX_BYTE_SIZE = 200 * 1024 * 1024;
 
+  private final boolean allowServersideTimestamp;
+
   private transient ImmutableList.Builder<com.google.bigtable.v2.Mutation> mutations =
       ImmutableList.builder();
 
   private int numMutations;
   private long byteSize;
 
   public static Mutation create() {
-    return new Mutation();
+    return new Mutation(false);
   }
 
-  private Mutation() {}
+  /**
+   * Creates Mutation object which allows setCell operation to set server side timestamp.
```

Comment:
Please make the docs a bit more scary:

`Creates Mutation object which allows setCell operation use set server side timestamp. This is dangerous because mutation will no longer be idempotent, which might cause multiple duplicate values to be stored in Bigtable. This option should only be used for advanced usecases with extreme care.`

Also, please add a `@BetaApi` to discourage usage

Reply:
Also, please add docs for the regular create